src/terminology.md

```diff
@@ -8,38 +8,54 @@ We group terms into related categories as follows:
 
 ## Protocol Concepts
 
-- *Assured Finality*: A protocol property that guarantees that final transactions cannot be reverted by that protocol. As with all protocol guarantees, a protocol assumes certain conditions must be met.
+<span id="definition-assured-finality"></span>**Assured Finality**: A protocol property that guarantees that final transactions cannot be reverted by that protocol. As with all protocol guarantees, a protocol assumes certain conditions must be met.
 
   Importantly, it is not feasible for any protocol to prevent reversing final transactions "out of band" from the protocol, such as if a sufficiently large and motivated group of users forks the network to include a specific new validity rule reverting transactions. For this reason, we forego the term "absolute finality" sometimes used in consensus protocol technical discussions.
 
 ## Protocol Components
 
-- *PoW+TFL*: the overall complete, integrated consensus protocol specified in this book.
-- *NU5*: the Zcash consensus protocol as of NU5.[^new-mainnet-precursors]
-- *PoW*: the PoW subprotocol within PoW+TFL. Note that this is a different consensus protocol from NU5 and encompasses more than narrow Nakamoto PoW consensus, including transaction semantics such as for shielded transfers.
-- *TFL*: the *Trailing Finality Layer* subprotocol within PoW+TFL.
+<span id="definition-pow-tfl"></span>**PoW+TFL**: the overall complete, integrated consensus protocol specified in this book.
+
+<span id="definition-nu5"></span>**NU5**: the Zcash consensus protocol as of NU5.[^new-mainnet-precursors]
+
+<span id="definition-pow"></span>**PoW**: the PoW subprotocol within PoW+TFL. Note that this is a different consensus protocol from NU5 and encompasses more than narrow Nakamoto PoW consensus, including transaction semantics such as for shielded transfers.
```

Comment:
I don't think "PoW" is a good name for this. Although most discussion of Bitcoin-like chains (including Zcash) conflates them as "L1", we have always tried to keep L0 (the best-chain protocol, which happens to use PoW) somewhat separate from L1 (transaction layer consensus rules).

I think we should define the followed concepts (I'm not wedded to these names for them):

* **BC**: the best-chain subprotocol within new-L0. This will initially be an adaptation of Zcash's existing low-level consensus layer that uses Proof-of-Work.
* **TFL**: the *Trailing Finality Layer* subprotocol within new-L0.
* **new-L0**: a low-level consensus protocol that integrates BC and TFL, but does not include transaction semantics.
* **new-L1**: the consensus layer that includes transaction semantics, such as for shielded transfers. In the proposed design, this will be built on top of new-L0.
* **NU5-L1**: the existing Zcash consensus protocol, at the same "L1" level, as of NU5.